frontend/static/js/form.js

```diff
@@ -268,6 +268,22 @@ PreloadController.prototype = {
   }
 };
 
+function issuesForSubdomainRemoval(status) {
+  var parentDomain = status.preloadedDomain;
```

Comment:
Nit: `ancestorDomain` might be more accurate, since this might be a grandparent, etc.